README.md

```diff
@@ -68,6 +68,11 @@ defmodule MyApp.Endpoint do
 end
 ```
 
+Wildcards can be used in the origins specification, by using regex:
+```elixir
+    origins: [~r{//foo.com}, ~r{//localhost}]
```

Comment:
I realized later, that I should match at the beginning, so this is the better way: ^https?:\/\/foo\.com